bugbug/tools/code_review.py

```diff
@@ -103,30 +103,21 @@ class LargeDiffError(Exception):
 {comments}"""
 
 
-PROMPT_TEMPLATE_FILTERING_ANALYSIS = """Please, double check the code review provided for the patch below.
-Just report the comments that are:
-- applicable for the patch;
-- consistent with the {target_code_consistency} source code;
-- focusing on reporting possible bugs, major readability regressions, or similar concerns;
-- filter out any descriptive comments;
-- filter out any praising comments.
+PROMPT_TEMPLATE_FILTERING_ANALYSIS = """Filter review comments to identify those that:
+- are consistent with the {target_code_consistency} source code;
+- focus on reporting possible bugs, major functional regressions, major issues, or similar concerns;
+- report major readability or design concerns.
 
-Do not change the contents of the comments and the report format.
-Adopt the template below as the report format:
-[
-    {{
-        "file": "com/br/main/Pressure.java",
-        "code_line": 458,
-        "comment" : "In the third code block, you are using `nsAutoStringN<256>` instead of `nsString`. This is a good change as `nsAutoStringN<256>` is more efficient for small strings. However, you should ensure that the size of `tempString` does not exceed 256 characters, as `nsAutoStringN<256>` has a fixed size."
-    }}
-]
-Do not report any explanation about your choice. Only return a valid JSON list.
+Exclude comments that:
+- only state obvious facts about the patch;
+- restate obvious facts about renamed variables or replaced code;
+- include praising, descriptive, or non-critical remarks
```

Comment:
This is based on https://github.com/mozilla/bugbug/pull/4827#issuecomment-2794557356:
```suggestion
- only describe the change;
- restate obvious facts like renamed variables or replaced code;
- include praising;
```